features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -65,6 +65,12 @@
         <feature>openhab-transport-mdns</feature>
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.freebox/${project.version}</bundle>
     </feature>
+
+     <feature name="openhab-binding-globalcache" description="GlobalCache Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
+        <feature>esh-transform-map</feature>
```

Comment:
Almost. It should actually be ["openhab-transform-map"](https://github.com/openhab/openhab-core/blob/master/features/openhab-esh-addons/src/main/feature/feature.xml#L51)!